app/models/article.rb

```diff
@@ -855,4 +861,12 @@ def detect_animated_images
 
     ::Articles::DetectAnimatedImagesWorker.perform_async(id)
   end
+
+  def contains_prohibitted_unicode_characters?(str:)
```

Comment:
nitpick: regex constant name uses prohibited (as far as I can tell the only correct spelling?) - these two methods should be `contains_prohibited_unicode_characters?` and `remove_prohibited_unicode_characters` (rather than prohibitted), and the call to these functions in validate title should be updated.

Reply:
My bad, will fix this a bit later today 🙈